### PR TITLE
Create a batch update to data_maps table

### PR DESCRIPTION
**Pull request by pzhao5**
Have a better way to convert a struct{} data with db tag into string format for update a table

### CODE REVIEW

**Comment by pzhao5** [comment]
Sure, how about I do a follow up on this issue. Added a TODO here

File: utils/db_utils.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+	"fmt"
+	"github.com/gobuffalo/pop"
+	"github.com/gobuffalo/pop/columns"
+	"github.com/gobuffalo/uuid"
+	"github.com/pkg/errors"
+	"reflect"
+	"sort"
+	"strconv"
+	"strings"
+)
```
Comment (rfornea) [inline utils/db_utils.go]: A lot of the variables in this file could use more descriptive names, to make it easier for other developers to be able to quickly look at know what the code is doing.  No need to worry about it for this PR, could address if when you push the PR to do the batch updates every X iterations.

Comment (pzhao5) [reply utils/db_utils.go]: Can you give me an example? I could update it.

Comment (rfornea) [reply utils/db_utils.go]: Yeah.  
c := columns.ColumnsForStructWithAlias(vPtr, m.TableName(), m.As)   
Could be something like:
columns := columns.ColumnsForStructWithAlias(vPtr, model.TableName(), model.As)

And something like this:  
m := pop.Model{Value: vPtr}
Could be something like this:
model := pop.Model{Value: vPtr}

Or like this:
xs = append(xs, fmt.Sprintf("'%s'", u.String()))  
It's not super obvious what it is.  I'm guessing those are strings with the column names, so maybe something like "columnName" or something.

**Comment by pzhao5** [comment]
I see. Will update it

**Pull request merged**